libs/model/src/models/community.ts

```diff
@@ -147,6 +147,19 @@ export default (
         type: Sequelize.TEXT,
         allowNull: true,
       },
+      community_indexer_id: {
+        type: Sequelize.STRING,
+        allowNull: true,
+      },
+      token_address: {
```

Comment:
This name is not descriptive enough. We have `PinnedTokens` and `LaunchpadTokens` so this should follow the same pattern e.g. `clanker_token_address`. Correct me if I am wrong but it seems that this is only used to make the job idempotent. If so, we should have a separate `ClankerTokens` table. If for some reason this doesn't make sense, then at the very least these new values should be removed from the default scope of the model.

Reply:
Also, it seems that the Clanker token should be pinned to the community i.e. create `PinnedToken`